tools/Makefile

```diff
@@ -32,3 +32,15 @@ tidy:
 	@ go mod tidy
 	git diff go.mod go.sum | cat
 	git diff --quiet go.mod go.sum
+
+ci-test-job: failpoint-enable
+	CGO_ENABLED=1 go test ./... -v -tags deadlock -race -cover || { $(MAKE) failpoint-disable && exit 1; }
+	$(MAKE) failpoint-disable
+
+failpoint-enable:
+	cd $(ROOT_PATH) && $(MAKE) failpoint-enable
+	go mod tidy
```

Comment:
Why do we need go mod tidy here?

Reply:
copy from another Makefile... seems like assume It's possible that enabling these failpoints might also involve adding dependencies which need `go mod tidy`

Reply:
This hypothesis doesn't seem to exist for the moment, so let's remove it for now